ethexe/db/src/database.rs

```diff
@@ -68,6 +68,7 @@ enum Key {
     SignedTransaction(H256) = 12,
 
     LatestData = 13,
+    Timelines = 14,
```

Comment:
Good idea, but maybe better to store here other protocol params also like max_validators, please add TODO here